core/object/script_language_extension.h

```diff
@@ -101,6 +101,19 @@ class ScriptExtension : public Script {
 	EXBIND1RC(bool, has_method, const StringName &)
 	EXBIND1RC(bool, has_static_method, const StringName &)
 
+	GDVIRTUAL1RC(Variant, _get_script_method_argument_count, const StringName &)
+	virtual int get_script_method_argument_count(const StringName &p_method, bool *r_is_valid = nullptr) const override {
+		Variant ret;
+		if (GDVIRTUAL_CALL(_get_script_method_argument_count, p_method, ret)) {
+			if (r_is_valid) {
+				*r_is_valid = ret.get_type() == Variant::INT;
+			}
+			return (ret.get_type() == Variant::INT) ? ret.operator int() : 0;
+		}
+		// Fallback to default.
+		return Script::get_script_method_argument_count(p_method, r_is_valid);
```

Comment:
Unsure if it should fall back when the method doesn't return a valid value here instead, like:
```cpp
		Variant ret;
		if (GDVIRTUAL_CALL(_get_script_method_argument_count, p_method, ret) && ret.get_type() == Variant::INT) {
			if (r_is_valid) {
				*r_is_valid = true;
			}
			return ret.operator int();
		}
		// Fallback to default.
		return Script::get_script_method_argument_count(p_method, r_is_valid);
```
It'd allow falling back, but also custom behaviour, but unsure

Reply:
I'm not sure I understand. The alternative in your comment is to allow returning values that aren't integers but that could be converted to integers?

Reply:
No it's to fall back to the default if it doesn't return an `INT`

Reply:
Then I'll add it to my next push